internal/pkg/describe/uri.go

```diff
@@ -178,6 +178,9 @@ func (d *BackendServiceDescriber) URI(envName string) (URI, error) {
 			if err != nil {
 				return URI{}, err
 			}
+			if !albURI.HTTPS && len(albURI.DNSNames) > 1 {
+				albURI, _ = albDescr.removeEnvDNSName(albURI)
```

Comment:
If we decided to ignore the error maybe we should do it in `removeEnvDNSName` and change its name to `bestEffort...` or add a comment to the function (since I guess it would only be used here)?

Reply:
good idea, fixed 🫡